start-apps

```diff
@@ -0,0 +1,2 @@
+#! /bin/bash
+docker-compose -f docker-compose.apps.yml up proposer
```

Comment:
Hey @signorecello, what about the Challenger?

Reply:
i think we can add a separate PR for challenger. Proposer is urgent